modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordXLSWriter.java

```diff
@@ -68,9 +68,20 @@ public DDMFormInstanceRecordWriterResponse write(
 			CellStyle headerCellStyle = createCellStyle(
 				workbook, true, "Courier New", (short)14);
 
-			createRow(
-				rowIndex++, headerCellStyle, ddmFormFieldsLabel.values(),
-				sheet);
+			Collection<String> values = ddmFormFieldsLabel.values();
+
+			if (values.size() > _COLUMNS_MAX_COUNT) {
+				if (_log.isWarnEnabled()) {
+					_log.warn(
+						StringBundler.concat(
+							"Form has ", values.size(),
+							" fields. Due to XLS file format limitations, ",
+							"the first ", _COLUMNS_MAX_COUNT,
+							" will be included in the exported file."));
```

Comment:
@cleydyr In this case, only the first `_COLUMNS_MAX_COUNT` will be included, right?

Reply:
That's correct. If there are more than 256 columns (fields), then the first 256 will be included and the rest will be discarded.

Reply:
Got it, @cleydyr! thanks